sdks/python/src/opik/evaluation/metrics/llm_judges/g_eval/metric.py

```diff
@@ -98,21 +102,22 @@ async def ascore(
             messages=request,
             logprobs=True,
             top_logprobs=20,
+            response_format=GEvalScoreFormat,
         )
 
         return self._parse_model_output(model_output)
 
     def _parse_model_output(self, content: ModelResponse) -> score_result.ScoreResult:
         try:
-            # original_score = content.choices[0].model_extra['logprobs']['content'][0]['token']
-            top_logprobs = content.choices[0].model_extra["logprobs"]["content"][0][
-                "top_logprobs"
-            ]
+            # Compute score using top logprobs
+            top_score_logprobs = content.choices[0].model_extra["logprobs"]["content"][
+                3
```

Comment:
1. What is "3" here? "0" could be somehow understood as the first item in the list, but "3" looks a bit strange :)
2. Let's add a docstring to this method with an example of the `content` dictionary. I think it will be way easier to understand this method's logic with an example of data format in front of the eyes.